caravel/assets/javascripts/SqlLab/main.css

```diff
@@ -28,6 +28,14 @@
     padding-top: 5px;
     padding-bottom: 5px;
 }
+
+.scrollbar {
+    position: relative;
+    width: 100%;
+    overflow-y: auto;
+    height: 100%;
```

Comment:
I found that adding ```overflow: hidden;``` to body in main.css can solve this problem, namely we just make the whole app not-scrollable, only two inner divs scrollable.